src/cljs/rems/fields.cljs

```diff
@@ -168,13 +168,18 @@
   (.. event -target -value))
 
 (defn text-field
```

Comment:
were you going to figure out a better name for `text-field`?

Reply:
If you were planning on doing it later along with the date-field refactor that was discussed earlier, that's fine too!

Reply:
i created a separate issue for refactoring this one

Reply:
looks like it's #2600, added a note comment about renaming `text-field` as well